apitest/features/sqitch-jwt.feature

```diff
@@ -15,15 +15,14 @@ Feature: Basic sqitch API interaction with JWT
         "message": "The system must be tested"
         }
         """
-        And  I send a POST request to "http://{api_server}/comments"
+        And  I send a POST request to "http://{buffalo_server}/api/comments"
         Then the response status should be "201"
-        And the JSON response root should be array
```

Comment:
I'm puzzled by this one, does it mean the response is no longer a list of comments? Looking at the code it seems like it is still a list of comments.

Reply:
@statik the POST is no longer returning a list as it only returns the last successful comment that was created. On the elm side it now just appends that single comment response to the list it already has.

Reply:
aha! that makes perfect sense, I missed this was a change only on post. nice work!